Library/Homebrew/.rubocop.yml

```diff
@@ -10,6 +9,8 @@ AllCops:
     - 'bin/*'
     - '**/Casks/**/*'
     - '**/vendor/**/*'
+    - 'extend/**/*_spec.rb'
+    - 'software_spec.rb'
```

Comment:
These aren't test files.

Reply:
No they are not but they are scanned by `rubocop-rspec` because of their filename.

Reply:
I'm not sure if its intentional that the files are scanned since the `rubopcop-rspec` requirement now is done in the tests directory but that's how the software behaves now. The behaviour is not the same as when I had the rspec requirement in the other `.rubocop.yml` file. Could it be a bug ?